frameworks/projects/MXRoyale/src/main/royale/MXRoyaleClasses.as

```diff
@@ -68,6 +68,7 @@ internal class MXRoyaleClasses
 	import mx.effects.Tween; Tween;
 	import mx.system.ApplicationDomain; ApplicationDomain;
 	import mx.rpc.http.HTTPService; mx.rpc.http.HTTPService;
+	import mx.rpc.http.mxml.HTTPService; mx.rpc.http.HTTPService;
```

Comment:
This line shouldn't be necessary.  The mx-royale-manifest will take care of it.

Reply:
Okay, i had missed adding mx-royale-manifest to compiler arguments.

Reply:
Removed.